lapce-data/src/data.rs

```diff
@@ -275,6 +309,109 @@ impl LapceData {
         }
     }
 
+    pub fn parse_path_line_column(path: &Path) -> Option<(PathBuf, LineCol)> {
```

Comment:
Oh yes, I almost forgot. Add inline
```rust
#[inline]
pub fn parse_path_line_column(path: &Path) -> Option<(PathBuf, LineCol)> {
```

Reply:
I have read somewhere that `#[inline]` improved performance if the function is around 3-6 lines. But if the function is longer, then `#[inline]` reduces performance. It's a trade-off between instruction cache pressure and quantity of function calls. That's how I understood it.

Reply:
1. `inline` is just a suggestion to the LLVM compiler to inline a function, it doesn't have to. As I remember `inline` handled by LLVM, `inline(always)` done in rustc MIR.
2. Your function isn't a big one.